sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -411,7 +416,10 @@ class HashJoinIterator(
       rightData: LazySpillableColumnarBatch): Option[JoinGatherer] = {
     withResource(new NvtxWithMetrics("hash join gather map", NvtxColor.ORANGE, joinTime)) { _ =>
       val maps = joinType match {
-        case LeftOuter => leftKeys.leftJoinGatherMaps(rightKeys, compareNullsEqual)
+        case LeftOuter | ExistenceJoin(_) =>
+          // existence join is implemented as a left outer join with GatherMap converted
+          // to the Boolean column "exists"
+          leftKeys.leftJoinGatherMaps(rightKeys, compareNullsEqual)
```

Comment:
An existence join is not like a left outer join.  A left outer join will replicate rows when there are multiple matches in the right table.